src/datahike/query.cljc

```diff
@@ -1244,6 +1390,7 @@
       true                                          (-post-process qfind)
       qreturnmaps                                   (convert-to-return-maps qreturnmaps)
       stats?                                        (#(-> context-out
+                                                          (update :settings dissoc :relprod-strategy) ;; <-- Don't return non-serializable data.
```

Comment:
Maybe we don't need to expose `:settings` at all and can add it to the line below so that it becomes `(dissoc :rels :sources :settings)`.